storage/index.md

```diff
@@ -7,6 +7,101 @@ has_children: true
 
 ## Storage
 
-- lance vs delta
-- parquet
-- optimization
+GlareDB is a compute-centered database engine, that separates storage
+from compute. This separation provides for a system that makes focuses
+on connections between data in different forms and at and higher
+levels of organization, and also provide a great deal of flexibility
+and support 
+
+While GlareDB is compute-centered, it has storage systems, and
+supports a number of different storage formats and paradigms. This
+multi-paradigm approach means that you can select the storage format
+that makes the most sense for your data, and your access patterns, and
+what's more you can bring different storage formats to different parts
+of your data as appropriate.
+
+Even though GlareDB is compute-focused, understanding storage formats
+and their impact on query performance and efficiency is crucial for
+designing powerful data infrastructure. 
+
+Where transactional systems use indexes and data normalization to meet
+performance targets, analytics systems take advantage data layout, and
+data formats to meet performance goals. In virtually all systems,
+there are trade-offs between storage compactness and query time, and
+understanding these trade offs will help you use GlareDB (and any!)
+system effectively.
+
+### Formats: Columns and Records 
+
+Internally, GlareDB uses the [Arrow] format to represent data, which
+is an in-memory columnar format. Columnar formats are great because
+they make operations over a single column or group of columns, like
+sumarizations and averages, more efficient and they make it easier to
+ignore columns that aren't relevant. 
+
+Formats like [parquet] and [lance] as well as engines like [Delta]
+and [Iceberg] that use parquet, are also columnar. These columnar
+storage formats are quite easy to translate into arrow internally, and
+store metadata in ways that make it possible to only read the required
+segments of a file off of the disk.
+
+Columnar formats have limitations: updates to a single record are
+often scattered throughout a file which makes them particularly
+expensive, similarly, and writing out columnar data may require
+additional resources, or be inefficient if writes happen in very small
+batches. For transactional workloads, columnar models make less sense. 
+
+This table outlines some of the data formats and sources that GlareDB
+supports and identifies which model. [n.b. Engines like MySQL and
+PostgreSQL have some multi-modality capability.]
```

Comment:
Is this what you mean? What does "n.b." mean? If MySQL and PostgreSQL can be multi-modal, should those have x's in both columns?

```suggestion
This table outlines some of the data formats and sources that GlareDB
supports and identifies the modality. [n.b. Engines like MySQL and
PostgreSQL have some multi-modality capability.]
```